Assets/Scripts/NPC/NpcMovement.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections;
+using Framework;
+using FrameWork.GridSystem;
+using UnityEngine;
+
+namespace NPC
+{
+    public sealed class NpcMovement : Movement
+    {
+        
+
+        private void Start() => StartCoroutine(MovingNpc());
+        public void MoveNpc(int points)
```

Comment:
```suggestion
        private void Start() => StartCoroutine(MovingNpc());
        
        public void MoveNpc(int points)
```
wel nette styling hebben